Travel-list/src/App.jsx

```diff
@@ -1,13 +1,92 @@
+import { useState } from "react";
 import "./App.css";
 import FormSection from "./components/FormSection";
 import Header from "./components/Header";
+import PackingList from "./components/PackingList";
+import Stats from "./components/Stats";
 
+let id = 0;
 function App() {
+  const [items, setItems] = useState([]);
+  const itemsCount = items.length;
+
+  const handleAddItem = (newItem) => {
+    setItems([...items, { ...newItem, id }]);
+    id++;
+  };
+
+  const handleDeleteItem = (id) => {
+    const index = items[id];
+    const updatedItems = [...items];
+    updatedItems.splice(index, 1);
+    setItems(updatedItems);
+  };
+
+  const packedItemsCount = () => {
+    return items.reduce((acc, item) => {
+      if (item.isItemPacked) {
+        return acc + 1;
+      } else {
+        return acc;
+      }
+    }, 0);
+  };
+
+  const totalPackedItemsCount = packedItemsCount();
```

Comment:
So use it at the bottom level.. after declaration of all the functions or directly call it inside the prop which you have to pass.